go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -46,9 +49,11 @@ type topologyWatcherFactoryFunc func(topoServer *topo.Server, hc discovery.Healt
 type throttlerFactoryFunc func(name, unit string, threadCount int, maxRate int64, maxReplicationLagConfig throttler.MaxReplicationLagModuleConfig) (ThrottlerInterface, error)
 
 var (
-	healthCheckFactory     healthCheckFactoryFunc
-	topologyWatcherFactory topologyWatcherFactoryFunc
-	throttlerFactory       throttlerFactoryFunc
+	healthCheckFactory       healthCheckFactoryFunc
+	topologyWatcherFactory   topologyWatcherFactoryFunc
+	throttlerFactory         throttlerFactoryFunc
+	topoCellsRefreshInterval = time.Minute
+	ErrFoundNoCells          = vterrors.New(vtrpcpb.Code_FAILED_PRECONDITION, "found no cells")
```

Comment:
New idea: instead of throwing this error I could fallback to the `target.Cell` (`*target *querypb.Target`) of the local node

Reply:
> New idea: instead of throwing this error I could fallback to the Cell from *querypb.Target (the local node)

I've implemented this idea. We'll fallback to the local cell and log and error that the fallback occurred